.github/workflows/build.yml

```diff
@@ -46,6 +46,10 @@ jobs:
           pattern: '*.sh'
           exclude: './.git/*,./vendor/*'
 
+      - name: Lint Check
+        run: |
+          sudo curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "${GOPATH}"/bin v1.40.1
```

Comment:
can we make it part of `make bootstrap`

Reply:
Do you mean to use `go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.40.1`? One small gotcha [golangci-lint docs](https://golangci-lint.run/usage/install/#install-from-source) are recommending to install via binary installation mode.

Reply:
So I can think of following way in make file
```sh
## Binaries required for make targets
## NOTE: We can add commands here which can't be installed
## via "go get"
## Example: golangci-lint tool document is not recommending
##			to use `go get <path>`. For more info:
##          https://golangci-lint.run/usage/install/#install-from-source
EXTERNAL_BINARIES=\
	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "${GOPATH}"/bin v1.40.1

.PHONY: bootstrap
bootstrap: controller-gen
	@for tool in  $(EXTERNAL_BINARIES) ; do \
		echo "+ Installing $$tool" ; \
		$tool \
	done
```
Is it good have something like above?

Reply:
make another end point for installating golangci and put that as dependent for bootstrap. There we can fire curl command.